app/src/main/java/org/groundplatform/android/ui/datacollection/tasks/polygon/DrawAreaTaskFragment.kt

```diff
@@ -96,6 +104,16 @@ class DrawAreaTaskFragment @Inject constructor() : AbstractTaskFragment<DrawArea
     viewModel.removeLastVertex()
 
     // Move the camera to the last vertex, if any.
+    moveToPosition()
+  }
+
+  private fun redoLastVertex() {
```

Comment:
For consistency with existing undo implementation, let's call it `restoreLastVertex`